spec/system/admin_orders_spec.rb

```diff
@@ -47,4 +47,23 @@
 
     expect(page).to have_content 'Payment status: completed'
   end
+
+  it 'allows admin to change shipment status to shipped' do
```

Comment:
Same here as in previous comment

Reply:
I'm not really sure how to set such spec up. If the user is not an admin, then visiting the admin root path there will be a redirection to the login page, and there is already an existing spec for this.
```
it 'does not allow non admin user to login to admin dashboard' do
    click_on 'Sign In'
    fill_in_sign_in_form_as user
    click_on 'Log in'
    visit admin_root_path

    expect(page).to have_content('You are not authorized')
  end
```
  Or should I do something like `expect(page).not_to have_css '#orders-table'`?  But this does not look like it belongs to a system spec.

Reply:
Maybe i should test `expect(page).not_to have_css '#orders-table'` in the request spec?

Reply:
We discussed this on slack :)